flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/table/CorrelateTest.scala

```diff
@@ -204,4 +203,18 @@ class CorrelateTest extends TableTestBase {
 
     util.verifyExecPlan(sql)
   }
+
+  @Test
+  def testConstantFunction(): Unit = {
```

Comment:
please add some tests about LEFT JOIN, RIHGT JOIN, FULL JOIN, and also with non-deterministic